examples/autoqasm/2_Expressing_classical_control_flow.ipynb

```diff
@@ -6,7 +6,7 @@
    "metadata": {},
    "source": [
     "# Expressing classical control flow in quantum programs\n",
-    "Quantum programs should be able to express classical control flow, such as if-else branches, for-loops and subroutines. In this notebook, you will learn to compose a quantum program with more than just gates! "
+    "In the getting-started notebook, you learn how to compose a basic quantum program with only quantum gates. A quantum program should also be able to express classical control flow, such as if-else branches, for-loops and subroutines. AutoQASM provides a Pythonic experience for writing control flow in your quantum program. With AutoQASM, you don't need to navigate through the thought process of first constructing the code blocks about control flow and then add them to your program. Instead, you write control flow the same way you would do in a Python program. The Pythonic experience in expressing classical control flow makes the quantum programs more readable, and makes it easier to build more complex programs."
```

Comment:
This is awesome!

Just one sentence confuses me: "With AutoQASM, you don't need to navigate through the thought process of first constructing the code blocks about control flow and then add them to your program." Is this supposed to be "without" control flow? "about control flow" doesn't make sense to me. This might also be confusing if the reader isn't very familiar with Qiskit control flow syntax